app/serializers/application.js

```diff
@@ -91,6 +91,10 @@ function normalize(payload) {
 }
 
 export default ActiveModelSerializer.extend({
+  init() {
```

Comment:
Did you noticed any issue because of this code not existed ?

Reply:
oh, I left that by mistake. Added it while testing things. There was no issue. I'll remove it : )